inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/interpolate.hpp

```diff
@@ -50,4 +63,13 @@ class InterpolateLayerTest : public testing::WithParamInterface<InterpolateLayer
     void SetUp() override;
 };
 
+class InterpolateLayerTestWithConfig : public testing::WithParamInterface<InterpolateLayerTestParamsWithConfig>,
```

Comment:
Maybe we'll leave only one test class and change instances in all plugins? 
I don't like the current solution as far it causes a lot of ridiculous copy-pasta

Reply:
I did it before last commit.
But I will also need to change tests in kmb plugin. Is is okay?

Reply:
Why not? But you will have to use product-config repo to trigger a build with corresponding branches

Reply:
Reverted changes with lots of copypasta